src/circular_buffer.jl

```diff
@@ -38,11 +38,7 @@ end
 @inline function _buffer_index(cb::CircularBuffer, i::Int)
     n = cb.capacity
     idx = cb.first + i - 1
-    if idx > n
-        idx - n
-    else
-        idx
-    end
+    return ifelse(idx > n, idx - n, idx)
```

Comment:
changing `if` to `ifelse` changes behavour.
Which could be a valid call in this case, but is out of scope for this PR which is about changing style
`?` does not change behavour
```suggestion
    return idx > n ? idx - n : idx
```

Reply:
true .. this was un-solicited change .. thanks for pointing it out